leasing/models/land_use_agreement.py

```diff
@@ -625,6 +630,28 @@ def __str__(self):
         )
 
 
+class LandUseAgreementReceivableType(models.Model):
+    """
+    In Finnish: Saamislaji
+    """
+
+    name = models.CharField(verbose_name=_("Name"), max_length=255)
+    sap_material_code = models.CharField(
+        verbose_name=_("SAP material code"), null=True, blank=True, max_length=255
+    )
+    sap_order_item_number = models.CharField(
+        verbose_name=_("SAP order item number"), null=True, blank=True, max_length=255
```

Comment:
Same here, `null=True` should be removed.

Reply:
fixed